.github/CONTRIBUTING.md

```diff
@@ -1,64 +1,65 @@
 
-Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct, or to ban temporarily or permanently any contributor for other behaviors that they deem inappropriate, threatening, offensive, or harmful.
+### Code contributions
+Any contributions to PlaceholderAPI's code should be done towards the `develop` branch. Targeting the `master` branch in your Pull request may get it closed without warning.  
+Additionally should you follow the `Google Codestyle` when changing the code and Javadoc.
 
-### Scope
+Some noteworthy points about Javadoc:
 
-This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+- `@param` tags are in a single line and any additional param tag after one don't have an empty line in between.  
+Example:  
+```java
+/**
+ * @param param1 description
+ * @param param2 description
+ */
+```
+- Tags follow the order `@param`, `@return`, `@since` and `@deprecated`
+- The `@deprecated` tag should mention the version of when the annotated object will be removed. This usually would be the next minor version.
 
-### Enforcement
+### Wiki/Javadoc contributions
+If your Pull request only targets the [wiki] or only changes the Javadoc of PlaceholderAPI (And not its actual code) should Pull requests target the `docs` branch.  
+The develop branch should only be targeted when you also change some of PlaceholderAPI's code.
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at our [Discord]. All
-complaints will be reviewed and investigated and will result in a response that is deemed necessary and appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
-
-Project maintainers who do not follow or enforce the Code of Conduct in good faith may face temporary or permanent repercussions as determined by other members of the project's leadership.
-
-### Attribution
-
-This Code of Conduct is adapted from the [Contributor Covenant][homepage], version 1.4,
-available at [http://contributor-covenant.org/version/1/4][version]
+## Code of Conduct
+We have a [Code of Conduct] to maintain a welcoming atmosphere in this project.  
+If your contributions go against the above linked Code of Conduct will we reserve the right t deny or revert your contributions and to permanently remove your access from this project.
```

Comment:
```suggestion
If your contributions go against the code of conduct, linked above, we reserve the right to deny or revert your contributions.
```

Reply:
I think `Code of Conduct` is fine in that capitalisation as I see no real reason why it should suddenly be all lowercase.